packages/core/autoResetRef/index.ts

```diff
@@ -0,0 +1,35 @@
+import { customRef, Ref } from 'vue-demi'
+
+/**
+ * Create a ref which will be reset to the default value after some time.
+ *
+ * @link https://vueuse.org/autoResetRef
+ * @param defaultValue The value which will be set.
+ * @param afterMs      A zero-or-greater delay in milliseconds.
+ */
+export function autoResetRef<T>(defaultValue: T, afterMs = 10000): Ref<T> {
```

Comment:
Maybe we could have `afterMs` be a ref so it can be changed?

Reply:
or more specifically set the type to MaybeRef, and then just use `unref` to access it so users can pass in either just a number or a ref

Reply:
I have changed `afterMs` to `MaybeRef`!